packages/contracts-bedrock/justfile

```diff
@@ -45,6 +45,22 @@ clean:
 test *ARGS: build-go-ffi
   forge test {{ARGS}}
 
+# Runs upgrade path variant of contract tests.
+# TODO: enforce that the op-deployer binary exists
+test-upgrade *ARGS: build-go-ffi
```

Comment:
I get an error when running this, from the `just build-go-ffi` step
```
$ just test-upgrade
cd ./scripts/go-ffi && go build
go: downloading github.com/ethereum-optimism/op-geth v1.101411.3-rc.1.0.20241126165630-b84907bf4d95
go: downloading golang.org/x/exp v0.0.0-20241009180824-f66d83c29e7c
go: downloading github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20241126105717-d31591e83048
go: downloading github.com/holiman/uint256 v1.3.2
go: downloading golang.org/x/crypto v0.28.0
go: downloading github.com/ethereum/go-verkle v0.1.1-0.20240829091221-dffa7562dbe9
go: downloading github.com/prometheus/client_golang v1.20.5
go: downloading github.com/urfave/cli/v2 v2.27.5
go: downloading golang.org/x/time v0.7.0
go: downloading github.com/stretchr/testify v1.10.0
go: downloading golang.org/x/sys v0.26.0
go: downloading github.com/fsnotify/fsnotify v1.8.0
go: downloading golang.org/x/sync v0.10.0
go: downloading golang.org/x/term v0.25.0
go: downloading github.com/golang-jwt/jwt/v4 v4.5.1
go: downloading github.com/klauspost/compress v1.17.11
go: downloading github.com/bits-and-blooms/bitset v1.13.0
go: downloading golang.org/x/text v0.19.0
go: downloading github.com/multiformats/go-multiaddr v0.14.0
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.5
package github.com/ethereum-optimism/optimism/packages/contracts-bedrock/scripts/go-ffi
        imports github.com/ethereum-optimism/optimism/op-node/rollup/derive
        imports github.com/ethereum-optimism/optimism/op-service/testutils
        imports github.com/ethereum-optimism/optimism/op-service/client
        imports github.com/ethereum-optimism/optimism/op-node/sources
        imports github.com/ethereum-optimism/optimism/op-node/rollup/derive: import cycle not allowed
error: Recipe `build-go-ffi-default` failed on line 27 with exit code 1
```

Reply:
to be clear, this is happening on my branch, but not develop?

Reply:
ok, just tested and it happens on develop as well, I will investigate—since it's not related to this PR I'll resolve this comment